### ISSUE TITLE
Check Angular-translate versions compatibility

### ISSUE THREAD

**Issue by robertochingon** [angular-translate/angular-translate#1845]
### Subject of the issue
How can I know which is the latest angular-translate version that works with AngularJS 1.0.7? Where can I check compatibilities?

### Your environment
Please read this stackoverflow post for details: https://stackoverflow.com/questions/50245390/performance-issues-in-angularjs-1-0-7-and-angular-translate

### Steps to reproduce
Tell us how to reproduce this issue. Please provide a working demo, you can use [this template](https://plnkr.co/edit/XorWgI?p=preview) as a base.

### Expected behaviour
Tell us what should happen

### Actual behaviour
Tell us what happens instead

**Comment by knalli** [comment]
@samal84  is right, that is obviously out-of-scope.

We are still running our test suite against different setups (local bower environment, scopes defined via `bower.json` files), see here https://github.com/angular-translate/angular-translate/tree/ee641d8f2c586364c48507af617a033ca33f1a1e/test_scopes Of course, only covered things within tests. There are obvious changes between 1.2 and 1.6.

For 1.7: I've pushed 080cd04, but we will what happened.

**Comment by samal84** [comment]
For now I am unblocked by using the [Selective dependency resolutions](https://yarnpkg.com/lang/en/docs/selective-version-resolutions/) feature of Yarn, and just added this to package.json:

```
"resolutions": {
  "angular": "1.7.0"
}
```

I'm not seeing any issues with angular-translate 2.17.1 when forcing it to use angular 1.7.0 like this.

**Comment by knalli** [comment]
Yeah, the tests have been run successfully too. Stick with this until released.

**Comment by knalli** [comment]
Let's summarize:

* We have 2018, we don't care about AJS 1.0.7.
* Support for AJS 1.7.0 is in angular-translate@2.18.0 officially (but 2.17 works also)

**Comment by lianee** [comment]
you should also look at bower-angular-translate-storage-cookie, there is still a wrong dependency in bower.json:
"angular-cookies": ">=1.2.26 <1.7"

**Comment by knalli** [comment]
Arh, sorry about that. 2.18.1 is your friend.
